aldryn_client/utils.py

```diff
@@ -89,8 +88,13 @@ def execute(func, *popenargs, **kwargs):
     if kwargs.pop('silent', False):
         if 'stdout' not in kwargs:
             kwargs['stdout'] = open(os.devnull, 'w')
-            # close file descriptor devnull after exit
-            kwargs['close_fds'] = True
+            if not sys.platform == 'win32':
```

Comment:
we have ``is_windows()``, so I guess it would make sense to use that everywhere.